bindata/v4.1.0/kube-apiserver/audit-policies-cm.yaml

```diff
@@ -129,3 +129,25 @@ data:
         resources: ["oauthclients"]
     # catch-all rule to log all other requests with request and response payloads
     - level: RequestResponse
+
+  user-requests.yaml: |
+    apiVersion: audit.k8s.io/v1beta1
+    kind: Policy
+    metadata:
+      name: UserRequests
+    omitStages:
+    - "RequestReceived"
+    rules:
+    - level: RequestResponse
+      resources:
+      - group: "oauth.openshift.io"
+        resources: ["oauthaccesstokens"]
```

Comment:
@stlaz do we get any value from other oauth resources?

Reply:
I think `oauthauthorizetokens` might be considered since those are a prerequisite to access tokens. You might want to avoid logging the `codeChallenge` field in that case.

@EmilyM1 a better commit message and a PR description would help here, I had trouble answering Stefan's question without knowing what's the purpose of this new policy, I had to look for the JIRA.

Reply:
We can limit audit logging to meta data level. That way codeChallenge shouldn't be visible.

Reply:
I used the Audit Policy in the Enhancement. If anything else needs to be Metadata level that isn't or otherwise switched up, just let me know here or in Jira.

I can add a description.